lib/dartino/dartino_util.dart

```diff
@@ -137,13 +137,12 @@ class _Dartino {
       var sdk = sdkFor(null);
       if (sdk != null && sdk.validate()) {
 
-        // Given a valid Dartino SDK, now validate the Dart SDK
-        if (sdkManager.sdk?.isValidSdk != true) {
+        // If there's no Dart SDK configured, use the one provided by Dartino.
+        if (sdkManager.noSdkPathConfigured) {
```

Comment:
This was causing issues on Phil's machine. I think the ideal solution might be to notify the user if we see an invalid sdk path? And give them an opportunity to correct it. We may want to unify the sdk location logic for dartlang, flutter, and dartino to make things a bit more deterministic. I suspect the order of things now depends on the timings of various Futures.